### PR TITLE
Update build_and_run.md

### PR DESCRIPTION
**Pull request by guoqing-noaa**

### CODE REVIEW

File: docs/build_and_run.md [hunk 1]
```diff
@@ -2,7 +2,9 @@
 `git clone --recursive git@github.com:rrfs2/rrfs-workflow.git`
 
 `cd sorc` and run the following command to build the system. This will take a very long time at the moment:
```
[the remark this answers is not in the document]
Comment (guoqing-noaa) [reply docs/build_and_run.md:4]: It still takes a long time if we build each component serially. 

I think we should be able to build different components in parallel, I just have not gotten time to test that yet.  
A simple way is to use the following three lines in build.all:
```
./build.wps  | tee ./log.build.wps &
./build.mpas | tee ./log.build.mpas &
./build.rdas | tee ./log.build.rdas &
```
Do you have time to test this? Thanks!

[the remark this answers is not in the document]
Comment (chunhuazhou) [reply docs/build_and_run.md:4]: I will test it and let you know how it goes. Thanks!

**Comment by guoqing-noaa** [comment]
Great. Thanks, @chunhuazhou

**Comment by guoqing-noaa** [comment]
I will go ahead to merge this.

**Pull request merged**